wremnants/datasets/datagroups2016.py

```diff
@@ -0,0 +1,103 @@
+from utilities import boostHistHelpers as hh, logging
+from wremnants import histselections as sel
+from wremnants.datasets.datagroups import Datagroups
+from wremnants.datasets.datagroup import Datagroup
+from wremnants.datasets import datasets2016
+
+logger = logging.child_logger(__name__)
+
+class Datagroups2016(Datagroups):
```

Comment:
This probably doesn't even need to be a class, could just be an instance of the datagroups class

Reply:
Done